src/test/TestUtils/TestProjectFixture.cs

```diff
@@ -310,15 +317,10 @@ public TestProjectFixture EnsureRestored(params string[] fallbackSources)
 
         public TestProjectFixture EnsureRestoredForRid(string rid, params string[] fallbackSources)
         {
-            var sourceProjectJson = Path.Combine(_testProject.ProjectDirectory, "project.json.template");
-            var targetProjectJson = Path.Combine(_testProject.ProjectDirectory, "project.json");
-
-            // apply RID to template
-            File.WriteAllText(targetProjectJson, File.ReadAllText(sourceProjectJson).Replace("{RID}", rid));
-
             if ( ! _testProject.IsRestored())
             {
-                RestoreProject(fallbackSources);
+                string extraMSBuildProperties = $"/p:TestTargetRid={rid}";
```

Comment:
Same thing here as the `RuntimeFrameworkVersion`.  Let's just use the regular property - `RuntimeIdentifier`.  There's no need to give a different name to this property.  It just adds confusion for new people coming in.